pkg/controller/vitesscluster/reconcile_cells.go

```diff
@@ -60,10 +60,22 @@ func (r *ReconcileVitessCluster) reconcileCells(ctx context.Context, vt *planets
 		},
 		UpdateInPlace: func(key client.ObjectKey, obj runtime.Object) {
 			newObj := obj.(*planetscalev2.VitessCell)
+			if vt.Spec.UpdateStrategy != nil && vt.Spec.UpdateStrategy.Type != nil {
```

Comment:
Usually we omit these nil checks when we know that defaulting will ensure they're always non-nil, for the sake of readability.

Reply:
Removed across the board.